### PR TITLE
fix(list-view): add wi state in list view

### PR DESCRIPTION
**Pull request by SMahil**
add wi state in list view

### CODE REVIEW

**Comment by fabric8cd** [comment]
@SMahil fabric8/fabric8-ui:SNAPSHOT-PR-2346-11 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2346-fabric8-planner.badger.fabric8.io

File: src/app/components/work-item-list-entry/work-item-list-entry.component.html [hunk 1]
```diff
@@ -60,7 +60,11 @@
                 almIcon
                 [iconType]="workItem.attributes['system.state']"
                 tooltip="{{workItem.attributes['system.state']}}"
-                placement="right"></span>
+                placement="right">
+              </span>
+              <span class="pull-left f8-wi__list-description-state-text padding-v-2">
+                {{ workItem.attributes['system.state'] }}
+              </span>
```
Comment (nimishamukherjee) [inline src/app/components/work-item-list-entry/work-item-list-entry.component.html]: state2
state1

The text needs to be centre aligned vertically.

**Comment by fabric8cd** [comment]
@SMahil fabric8/fabric8-ui:SNAPSHOT-PR-2346-12 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2346-fabric8-planner.badger.fabric8.io

File: src/app/components/work-item-list-entry/work-item-list-entry.component.html (hunk 1 above)
Comment (SMahil) [reply src/app/components/work-item-list-entry/work-item-list-entry.component.html]: fixed

**Comment by fabric8cd** [comment]
@SMahil fabric8/fabric8-ui:SNAPSHOT-PR-2346-13 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2346-fabric8-planner.badger.fabric8.io

**Comment by rgarg1** [comment]
@SMahil The changes look good!

**Comment by fabric8cd** [comment]
@SMahil fabric8/fabric8-ui:SNAPSHOT-PR-2346-15 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2346-fabric8-planner.badger.fabric8.io

**Pull request merged**